main.tf

```diff
@@ -25,11 +25,13 @@ resource "google_kms_key_ring" "key_ring" {
 }
 
 resource "google_kms_crypto_key" "key" {
-  count           = var.prevent_destroy ? length(var.keys) : 0
-  name            = var.keys[count.index]
-  key_ring        = google_kms_key_ring.key_ring.id
-  rotation_period = var.key_rotation_period
-  purpose         = var.purpose
+  count                         = var.prevent_destroy ? length(var.keys) : 0
+  name                          = var.keys[count.index]
+  key_ring                      = google_kms_key_ring.key_ring.id
+  rotation_period               = var.key_rotation_period
+  purpose                       = var.purpose
+  import_only                   = var.import_only
```

Comment:
Looks like we should bump min version in https://github.com/terraform-google-modules/terraform-google-kms/blob/master/versions.tf#L23 to 3.87

https://github.com/hashicorp/terraform-provider-google/blob/4c9d7290a7f19ba221d224ff5b95500b7a6f8ed9/CHANGELOG_v3.md#3870-october-04-2021

Reply:
Thanks for pointing that! Updated!